code/modules/reagents/newchem/drugs.dm

```diff
@@ -457,4 +458,173 @@ datum/reagent/fliptonium/overdose_process(var/mob/living/M as mob)
 					M.drop_r_hand()
 					M.adjustToxLoss(1)
 	..()
-	return
\ No newline at end of file
+	return
+
+//////////////////////////////
+//		Synth-Drugs			//
+//////////////////////////////
+
+//Ultra-Lube: Meth
+/datum/reagent/lube/ultra
+	name = "Ultra-Lube"
+	id = "ultralube"
+	description = "Ultra-Lube is an enhanced lubricant which induces effect similar to Methamphetamine in synthetic users by drastically reducing internal friction and increasing cooling capabilities."
+	reagent_state = LIQUID
+	color = "#1BB1FF"
+
+	process_flags = SYNTHETIC
+	overdose_threshold = 20
+	addiction_threshold = 10
+	metabolization_rate = 0.6
+
+/datum/chemical_reaction/lube/ultra
+	name = "Ultra-Lube"
+	id = "ultralube"
+	result = "ultralube"
+	required_reagents = list("lube" = 2, "formaldehyde" = 1, "cryostylane" = 1)
+	result_amount = 2
+	mix_message = "The mixture darkens and appears to partially vaporize into a chilling aerosol."
+
+/datum/reagent/lube/ultra/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	var/high_message = pick("You feel your servos whir!", "You feel like you need to go faster.", "You feel like you were just overclocked!")
+	if(prob(1))
+		if(prob(1))
+			high_message = "0100011101001111010101000101010001000001010001110100111101000110010000010101001101010100!"
+	if(prob(5))
+		M << "<span class='notice'>[high_message]</span>"
+	M.AdjustParalysis(-2)
+	M.AdjustStunned(-2)
+	M.AdjustWeakened(-2)
+	M.adjustStaminaLoss(-2)
+	M.status_flags |= GOTTAGOREALLYFAST
+	M.Jitter(3)
+	M.adjustBrainLoss(0.5)
+	if(prob(5))
+		M.emote(pick("twitch", "shiver"))
+	..()
+	return
+
+/datum/reagent/lube/ultra/overdose_process(var/mob/living/M as mob)
+	if(prob(20))
+		M.emote("ping")
+	if(prob(33))
+		M.visible_message("<span class = 'danger'>[M]'s hands flip out and flail everywhere!</span>")
+		var/obj/item/I = M.get_active_hand()
+		if(I)
+			M.drop_item()
+	..()
+	if(prob(50))
+		M.adjustToxLoss(10)
```

Comment:
Since IPCs don't feel pain, they don't get the "soft-crit" or "pain-crit" other races get at 100, instead only being subject to hard-crit at 150 like everyone else.

They also take bonus brute and burn damage and are killed by emp instantly.